src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -139,7 +139,7 @@ class WorkspaceSettingsPage extends React.Component {
                     </FixedFooter>
                 )}
             >
-                {hasVBA => (
+                {(isLoadingVBA, hasVBA) => (
```

Comment:
Maybe the current behavior is fine, but should we consider disabling the currency picker until we can confirm that the account does NOT have a VBA?